### ISSUE TITLE
Rayleigh_Ray2, PnPn failure

### ISSUE THREAD

**Issue by RonRahaman** [Nek5000/NekExamples#59]
The Rayleigh_Ray2, PnPn test fails with the following error:
```
1:     FAILURE: umax: Test value 0.005519951 exceeds target value 0.004549071 +/- 1e-05
```
Nek5000/Nek5000@ab953206e94737c92344bc69ddd05686feb0611e is the first bad commit

**Comment by stgeke** [comment]
The new test value is correct. The deviation comes from a code change in the lowMachSolver (extrapolation based on t^n instead t^n+1). I also suggest to use a delta of 1e-04.

**Comment by RonRahaman** [comment]
Fixed in PR #66
